qcodes/dataset/plotting.py

```diff
@@ -142,6 +142,10 @@ def set_axis_labels(ax, data, cax=None):
                        f'{len(data-1)} parameters, cannot plot '
                        f'that.')
             cbaxes.append(None)
+
+    if len(axes) != len(cbaxes):
+        raise RuntimeError("Non equal number of axes. Perhaps cbaxes is missing"
+                           "from one of the cases above")
```

Comment:
Detail: missing space between "missing" and "from".

Reply:
#1031